components/formats-gpl/src/loci/formats/in/ScreenReader.java

```diff
@@ -78,6 +78,7 @@
   // -- Fields --
 
   private String[] plateMetadataFiles;
+  private String[] excludeReaders;
```

Comment:
Hmm, OK. Too bad because it appears to fix idr0025 problems described in https://github.com/IDR/idr-metadata/pull/211.